### ISSUE TITLE
Does the data indicates if the scene is from a bedroom or a living room?

### ISSUE THREAD

**Issue by Jibanul** [ScanNet/ScanNet#135]
Does anyone know if the data tells you somewhere the source of a scene? whether the scene was from a bedroom or a living room. Figure 5 in the [paper](https://arxiv.org/pdf/1702.04405.pdf) shows a distribution of the scans organized by types (bedroom/living room). So there must be this information somewhere. Thank you.

**Comment by WentingXu3o3** [comment]
Yes, you can find a .txt file which indicates the room in every scene folder.

**Comment by WentingXu3o3** [comment]
But I don't see the room label assigned in the test set .txt file.(scene0707_00---0806_00)
Does anyone know where to find it?

**Comment by marios2019** [comment]
> But I don't see the room label assigned in the test set .txt file.(scene0707_00---0806_00) Does anyone know where to find it?

Did you manage to find the scene type for the test split?
